20150310-wp-core-contrib-day.md

```diff
@@ -0,0 +1,255 @@
+
+Change to the directory you just created when checking out the WordPress Core source, install the npm modules and run the default Grunt job.
+
+```
+cd wordpress-core
+npm install
+grunt
+```
+
+You should now see a couple of new directories in the WordPress Core check out, `node_modues` and `build`.
+
+![WordPress Core After Build](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/WordPress-Core-After-Build.png)
+
+From now on, any time you make a change to the WordPress source code found under the `src` directory you can simply run `grunt` at the checkout root again to update the contents of the `build` directory, which will be the root of your development site. We talk more about that later in the [Keep It Clean][] section.
+
+### Configure Your Local Site
+The WordPress Core source has a neat feature that lets you create your `wp-config.php` file in the root of the checked out source, and the site which is to be served from the `build` directory will still pick up the correct configuration.
+
+Copy the supplied `wp-config-sample.php` to `wp-config.php`and update it to include the database and database user details for a new MySQL database you have created locally.
+
+![WordPress Core With Config Files](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/WordPress-Core-With-Config-Files.png)
+
+While you're at it, why not copy the `wp-tests-config-sample.php` file to `wp-tests-config.php` and update it with the database and database user details for another database you've also created locally so that you can run the unit tests too? It's important to use a separate database for the unit tests as the database is completely refreshed during a unit tests run.
+
+Now you have a `wp-config.php` for your development site, it's time to set up a website such as http://wordpress-core.dev that uses the `build` directory as its document root. I use [MAMP Pro](http://www.mamp.info/en/mamp-pro/) for its simplicity and flexibility, but you should use whatever you're most comfortable with. Ideally use PHP 5.2.x in your development as this is the minimum supported version of PHP for WordPress so you don't want to use features from later versions of PHP.
+
+![WordPress Core Site Setup (MAMP PRO)](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/WordPress-Core-Site-Setup-MAMP-PRO.png)
+
+### Give It a Spin
+The moment of truth, make sure you can connect to your new WordPress development site and set it up.
+
+![WordPress › Installation](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/WordPress-%E2%80%BA-Installation.png)
+
+![Dashboard ‹ WordPress Core — WordPress](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/Dashboard-%E2%80%B9-WordPress-Core-—-WordPress.png)
+
+![WordPress Core | Just another WordPress site](https://s3.amazonaws.com/cdn.deliciousbrains.com/content/uploads/2015/03/WordPress-Core-Just-another-WordPress-site.png)
+
```

Comment:
You may want to suggest populating their install with some test content like http://wptest.io/. Pretty bad to test on an empty install.